plugins/plugin_generate/bin-templates/js-template-runtime/src/app.js

```diff
@@ -15,6 +15,27 @@ var HelloWorldLayer = cc.Layer.extend({
         var mainscene = ccs.load(res.MainScene_json);
```

Comment:
There is no res.MainScene_json by default in js_template_runtime, how is it added ?